README.md

```diff
@@ -1,5 +1,34 @@
 # code-challenges-javascript
-### Convert boolean To String:
+This repo is for Javascript code-challenges
+## Set up
+1) cd into the directory you want this repository in - clone it down
+```
+git clone https://github.com/aepaysinger/code-challenges-javascript.git
+```
+2) install devDependencies for testing
+```
+npm install --production
```

Comment:
What does `--production` do?  Didn't work for me
```
[6:27:59] ➜  code-challenges-javascript git:(main) npm install --production
npm WARN config production Use `--omit=dev` instead.

up to date, audited 1 package in 362ms

found 0 vulnerabilities
[6:28:07] ➜  code-challenges-javascript git:(main) npm test

> code-challenges-javascript@1.0.0 test
> jest

sh: jest: command not found
[6:28:28] ➜  code-challenges-javascript git:(main)
```
https://docs.npmjs.com/cli/v9/commands/npm-install